### PR TITLE
Miscellaneous cleanup - last version

### PR DESCRIPTION
**Pull request by BitYog**
Thoroughly checked version of the earlier pull request which failed.

### CODE REVIEW

**Comment by BitYog** [comment]
#615 Thoroughly checked before making the pull request. Reviews welcome.

**Comment by dhanushuUzumaki** [comment]
Nice Job 👍 @BitYog

File: src/pages/jquery/jquery-css-method/index.md [hunk 1]
```diff
@@ -1,45 +1,22 @@
 ---
-title: CSS Method
+title: jQuery CSS Method
```
Comment (systimotic) [inline src/pages/jquery/jquery-css-method/index.md]: The previous article seems to have more information. Can you take a look at how to best merge these?

File: src/pages/computer-science/data-structures/hash-tables/index.md [hunk 2]
```diff
@@ -1,15 +1,163 @@
 ---
 title: Hash Tables
 ---
-## Hash Tables
+### Hash Tables
```
Comment (systimotic) [inline src/pages/computer-science/data-structures/hash-tables/index.md]: Can you make this title `##`, and the `##` ones below it `###`?

File: src/pages/javascript/check-the-length/index.md [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ---
-title: Check the Length Property of a String Variable
+title: Check the Length
```
Comment (systimotic) [inline src/pages/javascript/check-the-length/index.md]: I think this title is clearer with `of a string` added to it, as this doesn't talk about checking the length of an array.

**Comment by BitYog** [comment]
Ok @systimotic Will open a PR as soon as I'm done.

File: src/pages/jquery/jquery-css-method/index.md (hunk 1 above)
Comment (BitYog) [reply src/pages/jquery/jquery-css-method/index.md]: Ok. Should I permanently remove the add-id-attributes articles; as it has very less value?

**Comment by BitYog** [comment]
Done with the changes. @systimotic 
@dhanushuUzumaki @invinciblycool

**Comment by BitYog** [comment]
Added the required commit. @systimotic

**Comment by BitYog** [comment]
@systimotic Resolved conflicts.

**Comment by BitYog** [comment]
@systimotic Haven’t seen any activity here since 2 days. Is my PR fine or do I need to make more changes.

**Comment by texas2010** [comment]
@BitYog next time, do not delete your fork repo. and please wait until anyone would review or something.  we have too many PR and few people review it.   thank you for being patient with us.

**Comment by BitYog** [comment]
@texas2010 OK. Will you be merging this PR?

**Comment by texas2010** [comment]
sorry about that.

**Pull request closed without merging**